### PR TITLE
fixing news header

### PR DESCRIPTION
**Pull request by D-Rosa99**
#### Description: 

I modified the header of the screen news and the form to get the image of every news
#### Linked issues:

fixes: [#93](https://trello.com/c/OLrI96bs/93-modify-the-header-in-news-screen-to-have-a-better-view-of-the-list)
#### Screenshots:

image

#### How to test:

- `git pull`
- `git checkout Fix/93-header-news-screen`
- Head over to screen news and check how does it look

### CODE REVIEW

**Review by Faustrox — APPROVED** [review]
LGTM

File: app/helpers/imagesSource.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export default function(dataList) {
   const data = dataList.news.map(item => ({
     ...item,
-    img: { source: { uri: Object.values(item.img)[1] } },
+    img: { source: { uri: Object.values(item.img)[3] } },
```
Comment (inteljudajocu) [inline app/helpers/imagesSource.js]: Why [3]?

**Review by JoseFcoEstrella — APPROVED** [review]

**Review by inteljudajocu — COMMENTED** [review]
Comments

File: app/helpers/imagesSource.js (hunk 1 above)
Comment (D-Rosa99) [reply app/helpers/imagesSource.js]: img is an array of image and the number 3 is the biggest one, the others are smaller

Comment (inteljudajocu) [reply app/helpers/imagesSource.js]: The dimension in that image is better suited to the changes in screen sizes between devices?

Comment (iocedano) [reply app/helpers/imagesSource.js]: I believe we need to improve this because not all image come with all sizes

Comment (iocedano) [reply app/helpers/imagesSource.js]: Something like
```js
/*
 Get Image Crop base on array
 
 @params img {Object} image crop object
 @params crops {Array} crops name order by the first name exist on image crop
@returns {String} the image url that match with the first crop name found in the image object
*/
const getImageCrop(img, crops) {
 const firstCropFound = crops.find((cropName) => img[cropName]);
 return firstCropFound ? img[firstCropFound] : img.master;
}
```

```js
img: { source: { uri: getImageCrop(item.img, ['120x67', '120x80', '120x86', '250x141', '250x167', '250x179']) } }
```

Comment (iocedano) [reply app/helpers/imagesSource.js]: 😓 they are different crops name for small images maybe with a regex could work.
but there is a better implementation than this

**Review by Faustrox — APPROVED** [review]
LGTM

**Review by inteljudajocu — APPROVED** [review]

**Pull request merged**